website/ruchy.php

```diff
@@ -477,7 +496,8 @@
         //include_once("timeline.php");
     } // for each numer in multilog
 
-    if ($kret_multilog == '1') {
+    if (isset($errors)) {
+    } elseif ($kret_multilog == '1') {
```

Comment:
```
+    if (isset($errors)) {
+    } elseif ($kret_multilog == '1') {
```
This pattern seems quite unusual for me, may you eventually add a comment or write it differently?